backend/remote-state/s3/backend_state.go

```diff
@@ -101,9 +101,29 @@ func (b *Backend) State(name string) (state.State, error) {
 
 	stateMgr := &remote.State{Client: client}
 
-	//if this isn't the default state name, we need to create the object so
-	//it's listed by States.
-	if name != backend.DefaultStateName {
+	// Check to see if this state already exists.
+	// If we're trying to force-unlock a state, we can't take the lock before
+	// fetching the state. If the state doesn't exist, we have to assume this
+	// is a normal create operation, and take the lock at that point.
+	//
+	// If we need to force-unlock, but for some reason the state no longer
+	// exists, the user will have to use aws tools to manually fix the
+	// situation.
```

Comment:
I see this comment that says the user needs to fix this manually, but given that I'd also expect to see somewhere an error message explaining that to the user. Is that pre-existing somewhere and I just can't find it?